.github/workflows/tests.yml

```diff
@@ -259,7 +295,7 @@ jobs:
           export MICROCLOUD_SNAP_CHANNEL="${{ matrix.microcloud }}"
 
           cd test
-          sudo --preserve-env=DEBUG,GITHUB_ACTIONS,MICROCLOUD_DEBUG_PATH,MICROCLOUDD_DEBUG_PATH,SKIP_VM_LAUNCH,SNAPSHOT_RESTORE,TEST_STORAGE_SOURCE,TESTBED_READY,BASE_OS,LXD_SNAP_CHANNEL,MICROCEPH_SNAP_CHANNEL,MICROOVN_SNAP_CHANNEL,MICROCLOUD_SNAP_CHANNEL ./main.sh setup
+          sudo --preserve-env=PATH,GOPATH,GOCOVERDIR,DEBUG,GITHUB_ACTIONS,MICROCLOUD_DEBUG_PATH,MICROCLOUDD_DEBUG_PATH,SKIP_VM_LAUNCH,SNAPSHOT_RESTORE,TEST_STORAGE_SOURCE,TESTBED_READY,BASE_OS,LXD_SNAP_CHANNEL,MICROCEPH_SNAP_CHANNEL,MICROOVN_SNAP_CHANNEL,MICROCLOUD_SNAP_CHANNEL ./main.sh setup
```

Comment:
Just one more here, what is the reason we have to pass the `PATH` and `GOPATH` variables here? 
The test suite grew the `collect_go_cover_files` func but it doesn't look to consume any executable out of those two paths?

Reply:
I added `PATH` and `GOPATH` for verbosity but upon further investigation I don't believe they are required - only `GOCOVERDIR` is required.

Reply:
Ok, can we get rid of `PATH` and `GOPATH` then?

Reply:
Sure thing!

Reply:
Done :)